model/src/main/kotlin/config/VulnerabilityResolutionReason.kt

```diff
@@ -31,6 +31,11 @@ enum class VulnerabilityResolutionReason {
      */
     CANT_FIX_VULNERABILITY,
 
+    /**
+     * The required fix (e.g. patch, update) has been successfully applied and verified, eliminating the vulnerability.
+     */
+    FIXED_VULNERABILITY,
```

Comment:
The semantics described in the commit message IIUC differs from the one in the KDoc. 
I believe KDOC should be aligned.

Reply:
Please explain where you feel is a mismatch, because I don't see any after re-reading.

For using this enum value the question boils down to "How can a vulnerability actually be fixed if it is reported by an advisor for that package?". And *one* of the answers is the example mentioned in the commit message: As vulnerabilities are looked up by package coordinates / purls, not by package content, the advisor cannot know anything about patches being applied while keeping coordinates the same.

Reply:
Basically, the advisor reported a false positive, because the identifier of the package is ambiguous. So, could we instead use the one currently named "false positive" ?


> Please explain where you feel is a mismatch

I found it deviates, because the KDoc does not speak about any package context.
For the use case where one has a third party depenceny with a vulnerability included in a product, I guess there is also the possibility that the "FIX" goes into product code, e.g. ensuring that the vulnerable code path is not used.